src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -991,9 +991,9 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                     }
                                 ")
                         .AddScriptBlockExpressionBinding(@"
+                                    Set-StrictMode -Off
```

Comment:
Maybe move to line 1186?

Reply:
There's no functional difference, but I tend to expect it to be at the top of the script.